### PR TITLE
Issues/232 update bwakit (resolves #232)

### LINKED ISSUE #232: need upgraded bwakit image
The TOPMed pipeline needs the latest version of bwakit and the latest version of samtools (which is not included in the latest bwakit).

### PR DESCRIPTION
**Pull request by tpesout**
needed to update bwakit version and installed very latest of samblaster

### CODE REVIEW

File: bwakit/test.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 class TestBWAKIT(unittest.TestCase):
 
     def test_docker_call(self):
-        out, err = check_docker_output(tool='quay.io/ucsc_cgl/bwakit:0.7.12')
+        out, err = check_docker_output(tool='quay.io/ucsc_cgl/bwakit')
```
Comment (cket) [inline bwakit/test.py:11]: This should be point to a particular tag

File: bwakit/Dockerfile [hunk 2]
```diff
@@ -2,11 +2,17 @@ FROM ubuntu
...
+RUN apt-get update && apt-get install -y curl bzip2 perl make g++
 
 # pull down bwa-kit
-RUN curl -L http://sourceforge.net/projects/bio-bwa/files/bwakit/bwakit-0.7.12_x64-linux.tar.bz2 \
-    | tar -xjC /opt/ 
+RUN curl -L http://sourceforge.net/projects/bio-bwa/files/bwakit/bwakit-0.7.15_x64-linux.tar.bz2 \
+    | tar -xjC /opt/
+
+# get latest version of samblaster and build
+RUN curl -L https://github.com/GregoryFaust/samblaster/releases/download/v.0.1.24/samblaster-v.0.1.24.tar.gz \
```
Comment (cket) [inline bwakit/Dockerfile:12]: Is samblaster needed by this tool?

**Review by cket — CHANGES_REQUESTED** [review]

File: bwakit/test.py (hunk 1 above)
Comment (tpesout) [reply bwakit/test.py:11]: the test fails unless I include the full tag (0.7.15--somegithash) which seems like a bad practice.  Without specifying a tag it should grab the latest build, which should be fine as long as "make test" actually does a build

File: bwakit/Dockerfile (hunk 2 above)
Comment (tpesout) [reply bwakit/Dockerfile:12]: samblaster (specifically this version which is later than the latest bwakit version) is needed by TOPMed

**Review by cket — APPROVED** [review]

**Pull request merged**